### ISSUE TITLE
Ошибка при экспорте расписания в календарь

### ISSUE THREAD

**Issue by fraggdiller** [0niel/university-app#410]
**Описание ошибки**
Нажимаю "экспорт в календарь", вылезает ошибка "Произошла ошибка при экспорте расписания"

**Воспроизведение ошибки**
Шаги воспроизведения:
1. Перейдите в 'Профиль'
2. Кликните на 'Экспорт в календарь'
3. Смотрите ошибку

**Ожидаемое поведение**
Расписание экспортируется в календарь

**Скриншоты**
![image](https://github.com/user-attachments/assets/0f118ac4-c7b1-4498-842d-c55b6e650336)

**Смартфон:**
 - Девайс: iPhone 15 Pro Max
 - OS: iOS 18.0
 - Версия MIREA app: 1.5.8

**Comment by 0niel** [comment]
> Версия MIREA app: 1.5.8

Привет. Сохраняется ли проблема в версии 1.5.9?

**Comment by VlaDkoR-280** [comment]
> > Версия MIREA app: 1.5.8
> 
> Привет. Сохраняется ли проблема в версии 1.5.9?

Да, я решил добавить расписание в календарь. В приложении не выдает никаких ошибок, однако в календаре ничего не появляется. Отмечу, что в списке активных календарей появляется календарь с названием группы, но я полистал и не нашел ни одной добавленной пары. Еще почему-то (уже достаточно давно) пропало расписание с виджета, то есть пишет, что пар нет, хотя в приложении все есть.

**Comment by phaepich** [comment]
В дополнение к этой ошибке пишу.
Криво работает экспорт в гугловский календарь. В самом календаре ни одна пара не появляется, но в виджете на рабочем столе появляются напоминалка, что, например, через 20 минут будет пара. С самуснговским календарем все работает хорошо. И было бы очень неплохо, если бы можно было экспортировать пары на выбор, а не все сразу. Pixel 7, Android 14

**Comment by necrosskull** [comment]
> В дополнение к этой ошибке пишу. Криво работает экспорт в гугловский календарь. В самом календаре ни одна пара не появляется, но в виджете на рабочем столе появляются напоминалка, что, например, через 20 минут будет пара. С самуснговским календарем все работает хорошо. И было бы очень неплохо, если бы можно было экспортировать пары на выбор, а не все сразу. Pixel 7, Android 14



Нужно в настройках включить локальную учетную запись "Университет"


<p align="center">
  <img src="https://github.com/user-attachments/assets/7108b273-ff08-43e5-929d-91259a1f86fa" width="200" />
  <img src="https://github.com/user-attachments/assets/c54422fd-23b1-4115-af81-0dddd176784b" width="200" />
  <img src="https://github.com/user-attachments/assets/5753fc75-a3d2-4e5d-b9af-aced4b5cb539" width="200" />
  <img src="https://github.com/user-attachments/assets/cdf9adfd-f1a7-4dc2-bcf7-6f738b0c4905" width="200" />
</p>
